README.md

```diff
@@ -289,7 +289,7 @@ These options are the same regardless of whether you use them with the
   (default: false)
 * **parallel-direct-writes=BOOL**: Allow the kernel to dispatch
   multiple, parallel (non-extending) write requests for files opened
-  with `direct_io=true` (if supported by the kernel)
+  with `direct_io=true` or `cache.files=off` (if supported by the kernel, added in version 6.2)
```

Comment:
In retrospect it is unclear to do so but my usage of direct_io here isn't the config option (which is deprecated) but the underlying FUSE option. And given recent changes coming FUSE will apply to O_DIRECT opened files as well it seems. So there isn't a non-technical or settings only way to indicate this fact. Probably best to just remove the direct_io mention all together.